### DISCUSSION TITLE
解锁码错误：不是有效的 Base-64 字符串

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#2542] [讨论]
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/12e57376-2576-4b90-a50d-2eb8f92eae35)
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/13992c08-cc94-45ae-abdc-65fd28d9c03e)
确认输入的没有问题

**Comment by hejiehao** [comment]
把你粘贴进启动器的解锁码发一下？

**Comment by MoYuan-CN** [comment]
见鬼 我把更新密钥塞进了解锁码窗口

**Reply by MoYuan-CN** [reply]
才发现
